.scalafmt.conf

```diff
@@ -0,0 +1,20 @@
+maxColumn = 112
```

Comment:
you can't make happy everyone, so it's better to make happier people who work with codebase more often
Eugene is a core contributor, so making him feel uncomfortable about writing/reading code because someone can read it from mobile is not fair, in my opinion. If scalajs core contributors read it on mobile, it's fine, they require it to be 80 as it's important to them
I'm not saying Eugene feels uncomfortable, he certainly can speak for himself, but my point is that most active contributors should have more weight on decisions like this

no intetion to make this a war, just a side-view :) :peace_symbol: